components/observations/ObservationsListView.tsx

```diff
@@ -47,45 +47,44 @@ interface ObservationFragmentWithPageIndexAndZoneAndSource extends ObservationFr
   source: SourceType;
 }
 
-export const ObservationsListView: React.FunctionComponent<ObservationsListViewProps> = ({center_id, requestedTime, initialFilterConfig, ...props}) => {
+export const ObservationsListView: React.FunctionComponent<ObservationsListViewProps> = ({center_id, requestedTime, additionalFilters, ...props}) => {
   const endDate = requestedTimeToUTCDate(requestedTime);
-  const originalFilterConfig: ObservationFilterConfig = useMemo(
-    () => ({
-      dates: {
-        value: 'past_week',
-      },
-      ...initialFilterConfig,
-    }),
-    [initialFilterConfig],
-  );
-  const [filterConfig, setFilterConfig] = React.useState<ObservationFilterConfig>(originalFilterConfig);
-  const [filterModalVisible, setFilterModalVisible] = React.useState<boolean>(false);
+  const originalFilterConfig: ObservationFilterConfig = useMemo(() => createDefaultFilterConfig(requestedTime, additionalFilters), [requestedTime, additionalFilters]);
+  const [filterConfig, setFilterConfig] = useState<ObservationFilterConfig>(originalFilterConfig);
+  const [filterModalVisible, setFilterModalVisible] = useState<boolean>(false);
   const mapResult = useMapLayer(center_id);
   const mapLayer = mapResult.data;
 
-  // when the initial filter inputs change, we should honor those
-  React.useEffect(() => {
+  // Filter inputs changed via render props should overwrite our current state
+  useEffect(() => {
     setFilterConfig(current => ({
       ...current,
-      ...originalFilterConfig,
+      ...additionalFilters,
     }));
-  }, [originalFilterConfig]);
+  }, [additionalFilters]);
 
-  const nacObservationsResult = useNACObservations(center_id, requestedTime);
-  const nacObservations: ObservationFragmentWithPageIndex[] = useMemo(
-    () => (nacObservationsResult.data?.pages ?? []).flatMap((page, index) => page.data.map(o => ({...o, pageIndex: index}))),
-    [nacObservationsResult],
-  );
-  const nwacObservationsResult = useNWACObservations(center_id, requestedTime);
-  const nwacObservations: ObservationFragmentWithPageIndex[] = useMemo(
-    () => (nwacObservationsResult.data?.pages ?? []).flatMap((page, index) => page.data.map(o => ({...o, pageIndex: index}))),
-    [nwacObservationsResult],
+  // From the 2023-24 season onward, we will only have NAC observations
+  const displayNWACObservations = center_id === 'NWAC' && endDate < new Date('2023-09-01');
+  const nacObservationsResult = useNACObservations(center_id, requestedTime, {
+    enabled: !displayNWACObservations,
+  });
+  const nwacObservationsResult = useNWACObservations(center_id, requestedTime, {
+    enabled: displayNWACObservations,
+  });
+
+  const observationsResult = displayNWACObservations ? nwacObservationsResult : nacObservationsResult;
```

Comment:
removed the code that deals with arrays of results - we only need to worry about one